command/sub8_missions/sub8/sub_singleton.py

```diff
@@ -158,11 +159,11 @@ def __init__(self, node_handle):
 
     @util.cancellableInlineCallbacks
     def _init(self):
-        self._moveto_action_client = yield action.ActionClient(self.nh, 'moveto', MoveToAction)
-        self._odom_sub = yield self.nh.subscribe('odom', Odometry)
-        self._trajectory_sub = yield self.nh.subscribe('trajectory', PoseTwistStamped)
-        self._trajectory_pub = yield self.nh.advertise('trajectory', PoseTwistStamped)
-        self._dvl_range_sub = yield self.nh.subscribe('dvl/range', Float64Stamped)
+        self._moveto_action_client = yield action.ActionClient(self.nh, '/moveto', MoveToAction)
```

Comment:
I changed it because I was having missions get stuck waiting for odom forever in sim, doesn't seem to be happening any more... huh. I'll switch it back